javascript/react/src/components/ArqueroDetailsList/renderers/DefaultColumnHeader.tsx

```diff
@@ -38,12 +55,38 @@ export const DefaultColumnHeader: React.FC<DefaultColumnHeaderProps> = memo(
 		const onSortClick = useDelegatedColumnClickHandler(column, onSort)
 		const onColumnClick = useDelegatedColumnClickHandler(column, onSelect)
 
+		const iconProps = useIconProps(validationResult)
+		const validationIconProps = useValidationIconProps(
+			iconProps,
+			validationResult,
+		)
+		const errorColumnMessages = useGetColumnValidationErrors(validationResult)
+
 		return (
 			/* eslint-disable jsx-a11y/mouse-events-have-key-events */
 			<div style={containerStyle}>
 				<When condition={iconName}>
 					<LeftIcon className={iconClassName} iconName={iconName} />
 				</When>
+
```

Comment:
I think this structure will result in two icons if the users configures one _and_ there are validation errors. Take a look at the Columns story for examples of other column header icons. It would be worth adding a user icon to your story to see if it looks weird adding another icon next to it (at minimum, I guess it will push the header text around). I am not sure if it is better to keep the user-defined icon and add the validation error one, or replace the user icon when there are errors.

Reply:
I tested it and it renders the icon in the same position, as the CSS uses a position: absolute property. So how do you want to solve this?